### DISCUSSION TITLE
Access data from another existing table connected by reference field.

### DISCUSSION THREAD

**Discussion by Settld-Admin** [AntlerVC/firetable#732] [Support / Q&A] [answered]
Is it possible to set up a column that can access data from a document in another collection that is referenced from a field in a row in the current table? For example I have an Inspections table that includes a reference to the user in the users collection that booked the inspection and a reference to the property in the properties collection that the inspection is for.  Is there a way to use that reference to access the value of a field in the document referenced to populate a new field in the current table?  For example, populating a cell in the same row of a new column with the user name from the referenced user document. 

Please see attached screenshot for further details
<img width="1790" alt="Screen Shot 2022-08-03 at 3 53 00 pm" src="https://user-images.githubusercontent.com/108853492/182536441-36fff431-a95e-4249-b8ed-01d3fc329a34.png">

**Comment by siurna** [comment, marked as answer]
I think you could do that by making a connector field and adding a cloud function to populate the field using your document id when you update "Property"

**Reply by Settld-Admin** [reply]
I was wondering if you could use a connector field but isn't that limited to returning a list of options from another table that a user can select from?  In this case the selection has already been made.  It's more like a derivative field only accessing values from another referenced table instead of the same table.

**Reply by Settld-Admin** [reply]
I appreciates Siura's initial response but can anyone else assist with my question?  At the moment I don't have a way forward.

**Reply by harinij** [reply]
@Settld-Admin In this case you can use the derivative column to get data from same table's row or any other collection as well. It is very flexible. For example, to get data from another collection you can use the `db` field to connect and retrieve data from another collection like so  `db.collection("collectionId")`

**Reply by SudgeFunday** [reply]
did u ever figure this out [Settld-Admin](https://github.com/Settld-Admin)?

**Comment by koyamasann** [comment]
I also this req, can anyone make a guide step by step?
